packages/esm-service-queues-app/src/patient-search/patient-scheduled-visits.component.tsx

```diff
@@ -82,6 +79,11 @@ const ScheduledVisits: React.FC<{
   const { queueLocations } = useQueueLocations();
   const [selectedQueueLocation, setSelectedQueueLocation] = useState(queueLocations[0]?.id);
 
+  // TODO: This needs fixing, we cannot just take the first queue and assume that is what is wanted
```

Comment:
This is the pre-existing behavior, I am just moving this declaration higher up in the file.  I have ticketed the need to change this here:  https://openmrs.atlassian.net/browse/O3-2955 but it is unrelated to this particular issue.